src/doc/rustc/src/platform-support/armv6k-nintendo-3ds.md

```diff
@@ -90,15 +98,24 @@ fn main() {
 
 Binaries built for this target can be run in an emulator (most commonly
 [Citra](https://citra-emu.org/)), or sent to a device through
-the use of a tool like devkitARM's `3dslink`.
+the use of a tool like devkitARM's `3dslink`. They may also simply be copied
+to an SD card to be inserted in the device.
 
-The `cargo-3ds` tool mentioned in "Building Rust programs" supports the use of
-`3dslink` with `cargo 3ds run`. The default Rust test runner is not supported, but
+The `cargo-3ds` tool mentioned in [Building Rust programs](#building-rust-programs)
+supports the use of `3dslink` with `cargo 3ds run`. The default Rust test runner
+is not supported, but
 [custom test frameworks](https://doc.rust-lang.org/beta/unstable-book/language-features/custom-test-frameworks.html)
 can be used with `cargo 3ds test` to run unit tests on a device.
 
-The Rust `library/std` test suite is not yet supported.
+The Rust test suite for `library/std` is not yet supported.
 
 ## Cross-compilation toolchains and C code
 
-C code can be built for this target using the aforementioned devkitARM toolchain.
+C code can be built for this target using the
+[devkitARM toolchain](https://devkitpro.org/wiki/Getting_Started).
+This toolchain provides `arm-none-eabi-gcc` as the linker used to link Rust
+programs as well.
+
+The toolchain also provides a `libc` implementation, which is required by `std`
```

Comment:
Maybe note that this libc implementation doesn't implement everything rustc expects?

Reply:
Sure, I can make sure I mention that that for the next commit I push.